### ISSUE TITLE
 some tickers are not working in version 1.47

### ISSUE THREAD

**Issue by ricardomerces** [ranaroussi/yfinance#119]
Ran, 

Some tickers are not working in version 1.47
Examples:
yf.Ticker("BRLEUR=X")
yf.Ticker("BTC-USD")

**version 1.45 working perfectly !**

version 1.47:

```
Traceback (most recent call last):
  File "cotacoes.py", line 6, in <module>
    print(bitcoin.info)
  File "/Users/rmerces/LABS/alexa-skills/.venv/lib/python3.7/site-packages/yfinance/ticker.py", line 122, in info
    return self.get_info()
  File "/Users/rmerces/LABS/alexa-skills/.venv/lib/python3.7/site-packages/yfinance/base.py", line 360, in get_info
    self._get_fundamentals(proxy)
  File "/Users/rmerces/LABS/alexa-skills/.venv/lib/python3.7/site-packages/yfinance/base.py", line 263, in _get_fundamentals
    for item in data['esgScores']:
KeyError: 'esgScores'

```

Running in Python 3.7

**Comment by ranaroussi** [comment]
Thanks for the heads up! I've fixed that in 0.1.48

**Comment by ricardomerces** [comment]
Thanks!
